src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.iOS/Airship.cs

```diff
@@ -261,6 +261,21 @@ public List<MessageCenter.Message> InboxMessages
             }
         }
 
+        private NSDate FromDateTime(DateTime dateTime)
```

Comment:
Note: I borrowed most of this logic from https://docs.microsoft.com/en-us/xamarin/cross-platform/macios/unified/overview